### ISSUE TITLE
 — Redirect to 404 

### ISSUE THREAD

**Issue by SwenSchaeferjohann** [upframe/upframe.io#115]
~Already implementable:  For when clicking on the LOGO users should get redirected to their respective mentor profile, not anymore to malik's. _Open in design backlog: Swen will come up with designs for A) landing page for non-logged-in users B) Redirection page for logged-in mentors, So we can redirect them properly (when they manipulate the url.) -- Once the designs are there, this issue will be updated, until then, dw._~

**UPDATE:** 
~- Everyone else (all those who are not logged in) should be redirected to Upframe Facebook page. (For this release)~
404 saving some time for now

**Comment by fabiofcferreira** [comment]
Why should you be redirected to your own profile page? Shouldn't the next page be the landing page?

If it's a non-logged user what should be the result? I could not understand by the italic notes you commented... So, could you please enlight me in that matter?

**Comment by SwenSchaeferjohann** [comment]
Eventually, we want it to be the 'landing page' / platform - right now as it's only the scheduling feature that we wanna release our 'landing page' will merely be smth like this: (up to change)
<img width="415" alt="landingpage01" src="https://user-images.githubusercontent.com/42959314/51991594-b53a1180-24ab-11e9-87ee-22a775511172.png">
How do mentors find their login page without manually typing url? Maybe adding this if its not much extra dev work:
<img width="392" alt="landingpage02" src="https://user-images.githubusercontent.com/42959314/51991732-00542480-24ac-11e9-82d2-38809b6810f1.png">

anyway, if the user is logged in already (mentor) we may just perma route them to their own profile, since this is as of now all we have

**Comment by SwenSchaeferjohann** [comment]
@fabiofcferreira Does that bring light into the matter? 🍡

**Comment by fabiofcferreira** [comment]
Yay! This is some nice material for discussion. The reason why we have removed the login button from the navbar was that we really wanted people to focus on the key feature (_scheduling meetups_).

But since, we already had some tests last 2w I guess it's safe to go forward and implement this login form in the navbar and the landing page itself.
Thoughts? @SwenSchaeferjohann @malikpiara @ulissesferreira

**Comment by SwenSchaeferjohann** [comment]
In that case above 'landing page' would replace the concurrent login site www.connect.upframe.io/login, right?

**Comment by SwenSchaeferjohann** [comment]
> if the user is logged in already (mentor) we may just perma route them to their own profile, since this is as of now all we have

What do u think about this @fabiofcferreira ?

**Comment by SwenSchaeferjohann** [comment]
And once she's logged out she can log in again via the 'landing page'

**Comment by SwenSchaeferjohann** [comment]
> In that case above 'landing page' would replace the concurrent login site www.connect.upframe.io/login, right?

Is it possible to just take this site and change the styling/integrate it?

**Comment by fabiofcferreira** [comment]
I guess that design sketch you have would enable us to remove the automatic redirection when you open [connect.upframe.io](connect.upframe.io) that forwards you to Malik's mentor public page.
This would also mean we could do the logo redirection (Mentors - Go to their public page; Non-logged users: Go to /)

That page would not replace /login but add another option for the users to login instead. You could login in the landing page or in /login.

We can redirect logged users (mentors only as of now) to their public page, but what about non-logged users? That's the only doubt I have.

Lastly, yes we can reuse most of the code from the /login page I guess.

**Comment by SwenSchaeferjohann** [comment]
Yes I agree 👍 Rather raw url manipulation should always be directed to "/" if its not a existing profile like "malik"


> We can redirect logged users (mentors only as of now) to their public page, but what about non-logged users? That's the only doubt I have.

I think it's fine to just direct unlogged users to '/' / the landing page

**Comment by malikpiara** [comment]
TLDR;
• When logged in the 404 goes to public profile of the user (mentor)
• When logged in clicking on the logo or going to `connect.upframe.io` should redirect to user public profile.

**Comment by SwenSchaeferjohann** [comment]
Had a chat with Malik, we agreed that as of now we can safe time in terms of 'non-logged in users':
Whenever those hit 404, let's redirect them to our Upframe Facebook page instead. 
We will integrate the landing page with another release later. 
What do you think?
@fabiofcferreira @ulissesferreira

**Comment by SwenSchaeferjohann** [comment]
Also check out issue #121 (Complementary for Logged in users).

**Comment by SwenSchaeferjohann** [comment]
Changed to 404; Actual landing page postponed

**Comment by ulissesferreira** [comment]
The other issue is already fixed I am now fixing the logged out situation :) will redirect to facebook
